indexer/src/main/java/org/uniprot/store/indexer/uniprotkb/converter/UniProtEntryCrossReferenceConverter.java

```diff
@@ -61,13 +67,7 @@ private void convertXref(List<UniProtKBCrossReference> references, UniProtDocume
                 case "unipathway":
                 case "ensembl":
```

Comment:
For ensembl: The others Ids are already searchable in default search
https://beta.uniprot.org/uniprotkb?query=ENSP00000263451%20AND%20accession%3AP21802

Reply:
Yeah, it is searchable. But we have to pass whole string. https://beta.uniprot.org/uniprotkb?query=ENSP000002%20AND%20accession%3AP21802 will not work(Notice the partial id).The behaviour is not same as searching something indexed in content(full_ci). I feel, the behaviour for some property value search will be different that others if we don't index all the property values in content field. But by adding ensembl ids or other ids again in content we are increasing the index size.

Reply:
https://beta.uniprot.org/uniprotkb?query=ENSP%20AND%20accession%3AP21802 does not work but
https://www.uniprot.org/uniprot/?query=ensp+accession%3Ap21802&sort=score works.